### ISSUE TITLE
Localization re-implementation

### ISSUE THREAD

**Issue by hishigami** [na1307/SimpleSixtarScorecard#2]
Hi! I'm the foreigner that randomly decided to translate the program into English (via edited machine TL...) once search was implemented. Sorry if the random branch appearance and this issue startle you since I also can't speak Korean at all ㅠㅠ
That aside, I'd also like to find a way to re-implement the translation that I made. I originally hacked a translation together when I thought the project's scope would stay small, but given its scope is evidentially increasing, hard-coding all of the program strings into English would quickly become a maintenance nightmare.
I've looked into a few ways to handle this, including satellite assembly, re-using the current resx files for some of the class files, and using a separate resource file/folder entirely. Any code comments/documents can likely be annotated right below the original rather than overwriting them outright as I originally did.
I've attached [the old class files with English](https://github.com/na1307/SimpleSixtarScorecard/files/14666638/SSS-EN.zip) prior to the recent commits for reference.

**Comment by na1307** [comment]
I have one question. How did you find out about this project?

**Comment by hishigami** [comment]
I lurked around on dcinside. I've mainly kept an eye on dc for Respect V related matters, but after recently picking up Sixtar in earnest I've kept my watch on the Sixtar minor gallery as well.
I ended up finding your project post [there](https://gall.dcinside.com/mgallery/board/view/?id=sixtargate&no=7225); it reminded me of [a similar scorecard project](https://github.com/hishigami/DJMAX-Record-Keeper) that I worked on as a hobby for Respect V a while back, though I've more or less moved on from it due to the rise of V-Archive and its own scorecard recorder.
Thing is, Sixtar is also in a very obviously niche position; and with the already minuscule EN playerbase the game has, I wanted to look into it since it's very liable to get swept under the rug even in that slow-moving gallery.
If this sounds really weird, it probably does in a lot of ways...
